### DISCUSSION TITLE
GitHub Pages 404 "Your billing plan does not support private pages"

### DISCUSSION THREAD

**Discussion by kgilov** [community/community#137193] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

hi,
a few days ago we lost access to our website via GitHub Pages. It has 404 and the Pages setup has a message that our plan does not support the private repo.
We do have Github Team plan ( educational) 'ucbepic' and Pages did have access to private repo to serve the HTTP content. 
In addition, we initiated new code deployment, but it fails as well.

Similar issues got resolved by re-applying the Pages permissions (allowing access to private again)

pages settings, error message:
Your organization's billing plan does not support private pages. To restore this page, change its visibility to public or contact your organization administrators for assistance.

**Comment by billynicholas47** [comment]
Reach out to GitHub's support team to inquire about the specific reason for the change and to request assistance in restoring access to private Pages. They can help you troubleshoot any billing or technical issues.

**Comment by xvpc** [comment]
Looks like you need to reach out to GitHub's support team to inquire about the specific reason for the change and to request assistance in restoring access to private Pages, They can help you troubleshoot any billing or technical issues.

**Comment by kgilov** [comment, marked as answer]
thank you. I did reach out to Support and they were able to restore the functionality.

cheers,
